edge-modules/DirectMethodReceiver/src/Program.cs

```diff
@@ -2,33 +2,25 @@
 namespace DirectMethodReceiver
 {
     using System;
-    using System.Globalization;
     using System.IO;
     using System.Net;
-    using System.Runtime.Loader;
     using System.Threading;
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Client;
-    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+    using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using ModuleLib;
 
     class Program
     {
-        public static int Main() => MainAsync().Result;
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("DirectMethodReceiver");
```

Comment:
I like using the Logger for consistency across our codebase.. but one concern I have here is that - the TempSensor (and other modules as well), are our samples for customers to write their modules. And I wonder if this is an overkill in that respect.. I don't want to give the impression that they need to use ILogger for their logging..

Reply:
will change back to use Console.Writeline in TempSensor sample.